app/parsers/bulkrax/csv_parser.rb

```diff
@@ -307,9 +308,33 @@ def retrieve_cloud_files(files)
     # export methods
 
     def write_files
-      CSV.open(setup_export_file, "w", headers: export_headers, write_headers: true) do |csv|
-        importerexporter.entries.where(identifier: current_record_ids)[0..limit || total].each do |e|
-          csv << e.parsed_metadata
+      require 'open-uri'
```

Comment:
what the new file structure looks like when csv's are being created for export
![image](https://user-images.githubusercontent.com/29032869/177659375-de287911-56be-4b0b-a5fe-d3ce762cb6bc.png)